### ISSUE TITLE
Extension: Ghani and Simmons (2004)

### ISSUE THREAD

**Issue by boyangqu** [uchicago-computation-workshop/ma_proposal_workshop_a1#5]

**Comment by boyangqu** [comment]
I’ve chose to explore the topic on the application of machine learning in auction pricing predictions. Since there is only few papers in this area, I’ve chose Ghani and Simmon’s study “Predicting the End-Price of Online Auction” that they conducted in Accenture Technology Labs. 

Ghani and Simmon explored the possibility and accuracy of using machine learning algorithms to predict the ending price of given auctions using feature known prior to the start of the auctions. The authors chose to manipulate a dataset from eBay auctions, which can be accessed conveniently in detail and is large in scale, as a representation of other online auctions. Using information about sellers, the auction item, recent auction results, and the auction, the author processed the attributes and created new attributes that are employed later in the machine learning models to predict the end prices. In more detail, they employed historical auction data from eBay, defined the set of features to be extracted, created meta-features that are derived from the initial set of features, and trained a classifier/extractor to use the training data to now extract features from unseen data.  The authors used a dataset containing all Palm Zire 21 auctions from the PDA category on eBay during a two-month period,  which consists of 1700 instances (1300 for training the models and 400 for testing).

After data collection, the author applied three machine learning algorithms and compared the results. The regression approaches, linear, polynomial with degree of 2 and 3, and CART (Classification & Regression Trees), predict the exact ending prices; the multi-class classification approach used decision trees and neural networks to predict the $5 price ranges; the multiple binary classification tasks approach is similar to multi-class classification approach, while at each level the task decide if end-price if higher than X in $5 intervals. The results showed that classification approaches yielded much more promising predictions than the regression approach. Furthermore, the multiple binary classification leads to a 96% accuracy in the $5 ending price range prediction, which is far better than the multi-class classification approach. 

This paper answered its research question very well, but in a limited way. It only explored auctions of one type of item on eBay website, and only three machine learning algorithms were used. The manipulation the authors did with the auction features to extract new features from unseen data excellent, and I think I can delve further into this technique during data analysis. With only one dataset and three algorithms, two similar to each other, the persuasiveness of the accuracy of machine learning in auction price prediction is limited, and I believe a more extensive study can be a good area to explore. 

A possible extension is to explore different auction datasets using more machine learning algorithms corresponding to the auction features. More specifically, I think I can use data from offline auction houses such as Sotheby’s and Christie's on different kinds of items, and then using a wider range of machine learning algorithms and compare their accuracy in predicting end prices of auction of different kinds of items.

References:
Rayid Ghani and Hillery Simmons. “Predicting the End-Price of Online Auctions “. In the Proceedings of the International Workshop on Data Mining and Adaptive Modelling Methods for Economics and Management (2004).
